eZ/Publish/Core/Search/Common/FieldValueMapper/Aggregate.php

```diff
@@ -61,7 +61,7 @@ public function canMap(Field $field)
     }
 
     /**
-     * Map field value to a proper Elasticsearch representation.
+     * Map field value to a proper Solr representation.
```

Comment:
Fixed in https://github.com/ezsystems/ezpublish-kernel/pull/1697/commits/fd99df72bdd5f7cc2a71f74527e869836905b4f4